bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -198,6 +186,10 @@ struct task_struct {
 	int pid;
 } __attribute__((preserve_access_index));
```

Comment:
Add `unsigned int tgid;` into `struct task_struct`.
It will get CO:RE relocated to the right offset in task_struct at load time.